content/en/community/hackathons/usoc23/baby-steps/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Session 02 - Baby Steps
+linkTitle: Bulding Unikernels
+weight: 5
+summary: |
+  We start presenting the modular architecture of Unikraft and how students can build their unikernels using the manual, `make`-based approach.
+  Expected time: 100 minutes
+---
+
+{{< readfile file="/community/hackathons/sessions/setup/content/intro.md" markdown="true" >}}
+
+
+## Reminder
+In the previous [session](community/hackathons/usoc23/overview), we have experimented with building and running several unikernels using the scripts provided in this [repository](https://github.com/unikraft-upb/scripts).
+Today, we will explore the modular architecture of Unikraft, and how we can leverage that in order to setup, configure, build and run our unikernels from scratch, without the use of scripts.
+
+Please make sure that you have installed all the necessary packages, as explained in the previous session.
+
+## Intro
+{{< readfile file="/community/hackathons/sessions/baby-steps/content/intro.md" markdown="true" >}}
+
+
+## Setting up Helloworld
+{{< readfile file="/community/hackathons/sessions/baby-steps/content/Setup_Helloworld.md" markdown="true" >}}
+
+
+## Preparing Nginx
+{{< readfile file="/community/hackathons/sessions/baby-steps/content/Setup_Nginx.md" markdown="true" >}}
+
+## Work items
+
+Now that we have finished building and configuring a couple of applications from scratch, you can try your hand at a few more others:
+* [`app-httpreply`](https://github.com/unikraft/app-httpreply)
+* [`app-redis`](https://github.com/unikraft/app-redis)
+* [`app-sqlite`](https://github.com/unikraft/app-sqlite)
+* [`app-python3`](https://github.com/unikraft/app-python3)
```

Comment:
I would add a brief list of what they need to select in the `menuconfig`. You can copy paste them from [this page](https://unikraft.org/docs/usage/make_build/#app-sqlite-for-x86_64). Just the list for every application, and mention that they can use the running scripts from the repositories so they don't get stuck on running.

Reply:
Hmm, yes, but that would give too much away, in my opinion : ) They are work items, after all. I would add a reference to the page you have just mentioned though, so they can do some digging on their own. What do you think?

Reply:
Ok, that's fair. Just add a link to the page, but mention that they should only look for config items, since the page uses the old directory layout.